compiler/hash-utils/src/printing.rs

```diff
@@ -3,6 +3,41 @@
 //! reporting errors.
 use std::fmt;
 
+/// This is used a wrapper around [`writeln!`] which integrates writing
+/// to a specified stream.
+///
+/// This macro conveniently unwraps the result of the write operation.
+#[macro_export]
+macro_rules! stream_writeln {
+    ($stream:expr, $($arg:tt)*) => {
+        use std::io::Write;
```

Comment:
These will sometimes produce unused import warnings

Reply:
No, it doesn't because this actually requires the `Write` trait to be in scope.

Reply:
Alright.,